hypothesis-python/tests/cover/test_stateful.py

```diff
@@ -31,7 +31,7 @@
 from hypothesis.database import ExampleDatabase
 from hypothesis.stateful import Bundle, GenericStateMachine, \
     RuleBasedStateMachine, rule, invariant, precondition, \
-    run_state_machine_as_test
+    run_state_machine_as_test, initialize
```

Comment:
The build is [currently complaining about import ordering](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/380715346). If you run `./build.sh format` from the repo root it will fix that for you.